### ISSUE TITLE
Parameters for Pascal VOC segmentation

### ISSUE THREAD

**Issue by bermanmaxim** [itijyou/ademxapp#14]
Hello,

I see you shared the training code for semantic segmentation and parameters used for the Cityscapes dataset for tuning from A1.

Could you please also share which settings where used for training voc_rna-a1_cls21_s8_ep-0001.params and voc_rna-a1_cls21_s8_coco_ep-0001.params from A1? Was it also with random cropping (or simple resize like at test-time), and is the number of epochs similar to the one of cityscapes?

Thank you

**Comment by huaxinxiao** [comment]
Same. 
I have tried to re-train the **VOC** model with 120 epochs (fixed learning rate). It seems the model is over-fitted after 90 epochs (77.65% on **VOC val** dataset). Then, I selected the No.90 epochs to continue training with linear learning rate (60 epochs). However, the performance doesn't improve much even worse after 20 epochs.

**Comment by zhui064** [comment]
@huaxinxiao I have used the pretrained model  voc_rna-a1_cls21_s8_coco_ep-0001.params to test , but in PASCAL VOC validation dataset,  I can only got about 76% mean IoU , which was reported 82.86% in parper and github. did u have similar question. and can u please tell me what is your training settings to train the model which got 77.65% on voc validation dataset?

**Comment by shipengai** [comment]
Hi, all.@bermanmaxim @huaxinxiao @zhui064.
How about the result on VOC val dataset of your own training . Is IoU beyond 79%?

**Comment by czzerone** [comment]
@huaxinxiao hello, can you provide your train command for the VOC model，did you train the model with the SBD dataset?
